stanford_person.install

```diff
@@ -2,18 +2,73 @@
 
 /**
  * @file
- * Install Tasks.
+ * File description.
+ *
+ * Long description.
  */
 
 use Drupal\views\Entity\View;
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
 
 /**
  * Implements hook_install().
  */
 function stanford_person_install() {
+  // Services.
+  $file_system = \Drupal::service('file_system');
+  $module_handler = \Drupal::service('module_handler');
+  $logger = \Drupal::logger('stanford_person');
+
   // Disable the core taxonomy_term view if it is enabled.
   $core_taxonomy_view = View::load('taxonomy_term');
   if ($core_taxonomy_view) {
     $core_taxonomy_view->disable()->save();
   }
+
+  // Save the file to public.
+  $destination = 'public://media/image';
+  $source = $file_system->realpath(
+    drupal_get_path("module", "stanford_person") .
+    "/lib/assets/img/stanford-person-default-profile-image.png"
+  );
+  $file_system->prepareDirectory($destination, FileSystemInterface::CREATE_DIRECTORY);
+  $saved = $file_system->copy($source, $destination, FileSystemInterface::EXISTS_REPLACE);
+
+  // We don't force the default content module to be enabled on install so we
+  // should check here if it exists or not so that it can do it's thing.
+  if (!$module_handler->moduleExists('default_content')) {
```

Comment:
I started by adding the image media as default content but I also do not want to enable default content in the .info.yml file so this is effectively duplicating that effort. Should I just pull out the default content exports?

Reply:
I suppose another option would be to update the template to default to the provided image instead of updating the field.

Reply:
the template seems like a decent approach. but it wouldn't help much in the future for custom views.

Reply:
Good point.